crates/sui/tests/protocol_version_tests.rs

```diff
@@ -100,6 +96,20 @@ mod sim_only_tests {
             .unwrap();
 
         monitor_version_change(&test_cluster, 2 /* expected proto version */).await;
+
+        // verify that the node that didn't support the new version shut itself down.
+        for v in test_cluster.swarm.validators() {
+            if !v
+                .config
+                .supported_protocol_versions
+                .unwrap()
+                .is_version_supported(ProtocolVersion::new(2))
+            {
+                assert!(!v.is_running());
+            } else {
+                assert!(v.is_running());
```

Comment:
nit, can you add a message to the asserts to identify each validator, in case it fails, to help track down what happened?